R/boundsPredprob.R

```diff
@@ -1,74 +1,86 @@
-#' Decision cutpoints for boundary (based on predictive probability)
+#' Decision cutpoints for boundary (based on predictive probability) for Decision 1 rule.
```

Comment:
Since you mention Decision 1 rule, better have a section in `@details` which references the definition of the Decision 1 rule